src/node-tools/chunks_over_ws_server.ts

```diff
@@ -15,49 +15,43 @@ const SERVER_PORT = 3000
 const POOL_SIZE = 4
 
 const initWorkerpool = async () => {
-  const world_demo_env = getWorldDemoEnvSettings()
-  const chunks_node_worker_pool = new WorkerPool()
-  await chunks_node_worker_pool.init(POOL_SIZE)
-  await chunks_node_worker_pool.loadWorldEnv(world_demo_env)
-  const chunks_scheduler = new ChunksPolling()
-  chunks_scheduler.chunksWorkerPool = chunks_node_worker_pool
-  return chunks_scheduler
+    const world_demo_env = getWorldDemoEnvSettings()
+    const chunks_node_worker_pool = new WorkerPool()
+    await chunks_node_worker_pool.init(POOL_SIZE)
+    await chunks_node_worker_pool.loadWorldEnv(world_demo_env)
+    const chunks_scheduler = new ChunksPolling()
+    chunks_scheduler.chunksWorkerPool = chunks_node_worker_pool
+    return chunks_scheduler
 }
 
 const initWsServer = async () => {
```

Comment:
Could everything that isn't production oriented be moved in a `test` folder outside the `src` ?